R/Quosure.R

```diff
@@ -0,0 +1,270 @@
+#' Reproducible class with environment and code.
+#'
+#' @slot code (`character`) to reproduce the environment
+#' @slot env (`environment`) environment which content was generated by the evaluation
+#' of the `code` slot.
+setClass(
+  "Quosure",
```

Comment:
I think tealEnv isn't bad or you could go further and call it a tealReactive

Reply:
I am thinking about the name and the key question is that if it is code + env or rather env + code? I think it's the latter one as it conceptually _just_ adds a code attribute to the environment. Therefore:
- envcode
- envc ("c" as a shortcut of "code")
- reprenv - this I don't like that much because the code might not necessary reproduce the environment (vide: password obfuscation; also code needs a (separate) environment on which it is evaluated - i.e. rlang' quosure)

(I am going to modify this post with new ideas)

Reply:
> tealReactive - as above + "reactive" indicates some shiny relationship which is not true

wishful thinking as a user of shiny I would love to be able to treat these objects in as similar way as possible to shiny reactives would make it super easier for people with shiny experience to use them